src/main/java/net/dv8tion/jda/api/managers/ChannelManager.java

```diff
@@ -160,7 +160,7 @@ default Guild getGuild()
     /**
      * Adds an override for the specified {@link net.dv8tion.jda.api.entities.IPermissionHolder IPermissionHolder}
      * with the provided raw bitmasks as allowed and denied permissions. If the permission holder already
-     * had an override on this channel it will be updated instead.
+     * had an override on this channel it will be replaced instead.
```

Comment:
Not that it is wrong, but why the change in vernacular?

Reply:
The reason here is that it seems like "updated" implies it only changes he set values, but in reality we completely replace the entity.

Reply:
Makes sense.